src/extras/MovieClip.js

```diff
@@ -170,8 +203,16 @@ MovieClip.prototype.gotoAndStop = function (frameNumber)
 
     this._currentTime = frameNumber;
 
-    var round = Math.floor(this._currentTime);
-    this._texture = this._textures[round % this._textures.length];
+    if (this._durations !== null)
+    {
+        this._lag = frameNumber % 1 * this._durations[this.currentFrame];
```

Comment:
`frameNumber % 1` returns always `0` if framenumber is an integer. `5.5 % 1` returns `0.5`, so you can stop the animation between frames.
Maybe the `_lag` property is a bit confusing, I can remove the `_lag` property and read it from the fraction part of `_currentTime` if you want, I just thought it may be imprecise